04/index.html

```diff
@@ -1,30 +1,53 @@
 <!DOCTYPE html>
 <html lang="pl">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
-</head>
-<body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
-            </li>
-            <li>
-                <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/global.css" />
+    <link rel="stylesheet" href="./styles/mobile.css" media="(max-width: 760px)" />
+    <link rel="stylesheet" href="./styles/desktop.css" media="(min-width: 761px)" />
+  </head>
+  <body>
+    <header>
+      <nav class="container">
+        <div class="container__mobile">
+          <a class="logo" href="/">LOGO</a>
+          <button type="button" onclick="showSidebar();" class="hamburger"></button>
```

Comment:
Zwróć uwagę, że ten button zajmuje jedynie 50% przestrzeni klikalnej dla wysokości. Ostatnia linia wychodzi poza. To powoduje, że przestrzeń między środkową linia, a ostatnią nie jest klikalny - trzeba to poprawić ;)